Buildings/Fluid/SolarCollectors/UsersGuide.mo

```diff
@@ -1,110 +1,252 @@
+</tr>
+<tr>
+<td> A </td>
+<td> Gross area </td>
+<td> Gross area </td>
+<td> Gross, aperture, or absorber area </td>
+</tr>
+<tr>
+<td> C </td>
+<td> (mDry*385, V) </td>
+<td> C </td>
+<td> (CDry or mDry*385, V) </td>
+</tr>
+<tr>
+<td> mperA_flow_nominal </td>
+<td> mperA_flow_nominal </td>
+<td> mperA_flow_nominal </td>
+<td> m_flow_nominal/A </td>
+</tr>
+<tr>
+<td> dp_nominal </td>
+<td> / </td>
+<td> / </td>
+<td> dp_nominal </td>
+</tr>
+<tr>
+<td> b0, b1 </td>
+<td> IAM({0°,10°,...,90°}) </td>
+<td> IAM({10°,20°,...,90°}) </td>
+<td> IAM(angle)-plot </td>
+</tr>
+<tr>
+<td> y_intercept, slope (ASHRAE93) </td>
+<td> y_intercept, slope </td>
+<td> / </td>
+<td> / </td>
+</tr>
+<tr>
+<td> IAMDiff, eta0, a1, a2 (EN12975) </td>
+<td> IAM(50°), eta0, a1, a2 </td>
+<td> Kd or IAM(50°), eta0, a1, a2 </td>
+<td> IAM(50°), eta0, a1, a2 </td>
+</tr>
+</table>
+</p>
+<p>
+Some extra important remarks regarding the performance data:
+<ul>
+<li>
+Different areas can be defined for a solar collector: the gross, absorber, and
+aperture area.
+The performance parameters of the solar collector vary depending on the area
+for which they are defined.
+Therefore, the performance parameters used in the data record should match the
+area that is used.
+</li>
+<li>
+When the thermal capacity of the solar collector without fluid is not known,
+the thermal capacity is calculated based on the dry mass of the collector
+and the specific heat capacity of copper (<i>385 J/kg/K</i>).
+</li>
+<li>
+All data sources report a nominal mass flow rate (per unit area of collector),
+but only SPF reports a corresponding nominal pressure drop.
+If a specific collector is used that is not included in the SPF database, one
+can likely find this via the manufacturer (website or on request).
+Some examples of (<code>mperA_flow_nominal</code>, <code>dp_nominal</code> can
+be found in <a href=\"modelica://Buildings.Fluid.SolarCollectors.Data\">
+Buildings.Fluid.SolarCollectors.BaseClasses</a>.
+</li>
+<li>
+Pressure drops depend on the medium that is used in the collectors.
+If the modelled solar thermal collector uses a different medium than the medium
+that was used to determine the nominal pressure drop in a data sheet, one should
+therefore correctly take this into account (e.g. using an empirical correction
+factor).
+</li>
+<li>
+The relation between the incidence angle modifier (IAM) and incidence angle
+<code>&theta;</code> is calculated using cubic splines and measurement data
+provided in the data sheets.
+</p>
+<p>
+As reported in the Energyplus Engineering Reference, this relation is only valid
+for incident angles of 60 degrees or higher. However, as opposed to the EnergyPlus
+approach, the IAM is not set to 0 for incident angles greater than 60 degrees
+(see Buildings
+<a href=\"https://github.com/lbl-srg/modelica-buildings/issues/785\">
+issue 785</a>).
+Therefore, the simulation results for incident angles greater than 60 degrees
+should be interpreted with care.
```

Comment:
I don't understand this. Is this old text that can be removed as we now use cubic splines?

Reply:
This indeed had to do with the old implementation and can now be removed.